TRB_label_assist/models.py

```diff
@@ -880,13 +922,13 @@ def set_params(self, params):
 
         return self
 
-    def fit(self, train_df):
+    def fit(self, train_df,train_entry_list=None):
```

Comment:
shouldn't this be `unused` since we were not passing in the entries earlier?

Reply:
In case when `start_cluster_model` is `RefractoredNaiveCluster` ( which is based on greedy similarity binning), we use the list of entry type data while fitting. However if `start_cluster_model` is DBSCAN, train_df is used. This is based on the value of `self.alg` parameter passed.